pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -80,26 +80,39 @@ func ConvertIngressV1alpha3(ingress v1beta1.Ingress, mesh *meshconfig.MeshConfig
 
 	for i, tls := range ingress.Spec.TLS {
 		if tls.SecretName == "" {
-			log.Infof("invalid ingress rule %s:%s for hosts %q, no secretName defined", ingress.Namespace, ingress.Name, tls.Hosts)
-			continue
+			gateway.Servers = append(gateway.Servers, &networking.Server{
+				Port: &networking.Port{
+					Number:   443,
+					Protocol: string(protocol.HTTPS),
+					Name:     fmt.Sprintf("https-443-ingress-%s-%s-%d", ingress.Name, ingress.Namespace, i),
+				},
+				Hosts: tls.Hosts,
+				Tls: &networking.ServerTLSSettings{
+					HttpsRedirect:  false,
+					Mode:           networking.ServerTLSSettings_AUTO_PASSTHROUGH,
```

Comment:
```suggestion
					Mode:           networking.ServerTLSSettings_PASSTHROUGH,
```

Auto passthrough is very different, its for multinetwork